x-pack/plugins/fleet/server/routes/app/index.ts

```diff
@@ -142,7 +160,26 @@ const serviceTokenBodyValidation = (data: any, validationResult: RouteValidation
   return ok({ remote });
 };
 
-export const registerRoutes = (router: FleetAuthzRouter) => {
+export const registerRoutes = (router: FleetAuthzRouter, config: FleetConfigType) => {
+  const experimentalFeatures = parseExperimentalConfigValue(config.enableExperimental);
+
+  if (experimentalFeatures.useSpaceAwareness) {
+    router.versioned
+      .post({
+        path: '/internal/fleet/enable_space_awareness',
```

Comment:
Question: Does this route need authz access tags?

Reply:
We have our own logic for doing authentication in Fleet that is done via the configuration
```
 fleetAuthz: {
          fleet: { all: true },
        },
```